pkg/topkg.ml

```diff
@@ -144,7 +148,12 @@ end
 module Exts : Exts = struct
   let interface = [".mli"; ".cmi"; ".cmti"]
   let interface_opt = ".cmx" :: interface
-  let library = [".cma"; ".cmxa"; ".cmxs"; ".a"]
+
+  let platform_lib_extension =
```

Comment:
With the new name this should fit in the 80 columns of one line. If it does  please do so and stick it to the other defs.